src/app/callback/route.ts

```diff
@@ -43,6 +43,7 @@ export async function GET(request: NextRequest) {
 
       return response;
     } catch (error) {
+      console.error(error);
       return NextResponse.json(error);
```

Comment:
```suggestion
      return NextResponse.json({
        error: error instanceof Error ? error.message : String(error),
      });
```

Should we also update the response to correctly output an error message?

Reply:
Love it.